README.md

```diff
@@ -213,3 +213,32 @@ Applying `docker-bakery` is quite simple. Take a look [here](https://github.com/
 <a id="limitations"></a>
 # Limitations
 At the moment docker multi-stage builds are not fully supported. Only the first line `FROM` is taken into consideration when determining the parent of the image and its place in the hierarchy. 
+
+## Command copy-images-hierarchy
```

Comment:
Description of this command should not be a subsection of #Limitations.
Also the TOC table should be recreated.

Reply:
done